modules/ocf_mirrors/files/process-mirrors-logs

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+import argparse
+import os
+from datetime import date
+
+import pymysql
+
+OCFSTATS_PWD = os.environ['OCFSTATS_PWD']
+MIRRORS_PATH = '/opt/mirrors/ftp'
+LOG_PATH = '/var/log/apache2'
+LOG_NAME = 'mirrors.ocf.berkeley.edu_access.log'
```

Comment:
I think I forgot to mention this before, but the reason why I was using `mirrors.ocf.berkeley.edu_access.log.1` instead of this was because the logs get rotated at around 6 AM, so if you check them once a day, unless you check very close to when they are rotated, you lose some data. For example, this is currently checked at midnight, so all the data from midnight to around 6 AM is currently lost since it is rotated before the next midnight.